### ISSUE TITLE
Add `bullet list` component

### ISSUE THREAD

**Issue by B3nz01d** [Orange-OpenSource/ouds-ios#513]
# Description

**As a** user of the library
**I want** to be able to use the component
**So that** I can add and configure in my own projects

**As a** user of the library
**I want** to be able to use the component
**So that** I have access to the documentation on how to use it

**As a** user of the Design System Toolbox
**I want** to access the component 
**So that** I can understand how to use it:
* while using the demo app I can interact with the available options and observe the changes on the component itself
* I can see the code associated to the component customization
* I can copy and paste the code from the app based on the selected customization
* or when looking into the code of the app itself, I would also understand how to use it

**Comment by B3nz01d** [comment]
@ludovic35 :
* I had the same feedback as the first item from Jerome.
* The second issue is that when a long text is provided the position of the icon should be aligned with the first line of text

**Comment by pylapp** [comment]
> side question, if you where to use a webview integration within the app would it be the responsability of the parser we provide to make this bullet list appearance or do we consider it's the html's provider responsability?
> 

@B3nz01d Il faut voir quel est le besoin. De mon point de vue, à partir du moment où on utilise une _web view_ on délégue le style à ce qui est affiché par la _web view_. Dit autrement on ne fait pas notre une responsabilité que l'on n'a pas à prendre.

De plus, que ce soit en terme de performances que de sécurité, et de maintenance, injecter du CSS ou du JS dans des _web view_ est une truc mauvaise idée, qui dégradera l'expérience utilisateur avec de potentiels glitchs ou retard à l'affichage.

Si la _web view_ vu un rendu particulier des _bullet list_ Elle devra en toute logique prendre la lib web OUDS.

**Comment by pylapp** [comment]
@pya35 @Aniort Je me pose une question avec Voice Over : comment bien vocaliser les entrées dans une liste ordonnée ou non ?

Par exemple à date on vocalise le numéro ou la lettre, avec une pause marquée, puis le texte.
Est-ce que l'on devrait faire tout en une fois ? Vocaliser en plus quelque chose "autours" du numéro ou de la lettre pour bien faire comprendre ce que c'est un indice de position dans la liste ?

**Comment by jerome-regnier** [comment]
Design Review Figma :
https://www.figma.com/design/yKtkEY1DOl7YDvmt7jq0QF/-OUDS-Core-Lib--Review?node-id=70829-5574&t=8BMvTW4FHsb1cGSY-1

👌 OK Design

**Comment by pylapp** [comment]
Clairement je ne suis absolument pas d’accord.

On parle d’une liste ordonnée ; de fait les éléments doivent être ordonnés. Utiliser des puces en dernier niveau va casser complètement la logique. De plus, quid de l’accessibilité avec la vocalisation des puces qui justement donnent une information sur l’ordre des éléments ?

Il serait bien plus intelligent de combiner des doubles lettres par exemple.

Avec tout le respect que j’ai pour Hamza et Ahmed-Amine, autant leur usage de la langue arabe est précieux autant ce n’est pas à eux de décider unilatéralement ce qui doit être fait.

L’accessibilité risque d’entre prendre un coup, et comme facilement un tiers des pays Orange parlent arabe on ne peut se permettre ce genre de prise de risque non mesurée.

J’ajouterai aussi que ce comportement est incohérent : pourquoi soudainement dans une langue on passe par des puces en 3ème niveau pour une seule langue ? À nous de trouver une solution pertinente et celle proposée ne l’est aucunement.

Pourquoi pas par exemple :
- doubler les lettres (mais quid du cas où listes en amont sont plus longue que l’alphabet doublant alors déjà les lettres)
- utiliser les lettres et les mettre en gras 

Je refuse de suivre la solution d’une puce qui va contre toute logique. Ma solution d’utiliser des lettres avec une graisse différentes, ou de répéter les lettres est meilleure.

**Comment by pylapp** [comment]
📣 **New TestFlight ALPHA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox ALPHA (513)**
- Version: **1.2.0**
- Build number: **1770196166**
- Build tag: **37c4af3**


cc @B3nz01d (product owner)
cc @pylapp @ludovic35 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (a11y team)

_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **1.2.0**
- Build number: **1770933877**
- Build tag: **9a21820**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New release available** 🚀

Please, get the [new release from GitHub](https://github.com/Orange-OpenSource/ouds-ios-design-system-toolbox/releases/tag/1.2.0) and wait for App Store update to test it!

- Display name: **Design System Toolbox**
- Version: **1.2.0**
- Build number: **1770992443**



_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_
